services/Directory/data/FilterList.json

```diff
@@ -14182,12 +14182,20 @@
     "name": "Yuki's uBlock Japanese filters - Annoyances"
   },
   {
-    "description": "Enhance anti-scam capability of 280blocker for Japanese mobile site (ABP format) by utilizing advanced capability of AdGuard/uBlock Origin.",
+    "description": "Enhance anti-scam capability of 280blocker adblock filter by utilizing advanced capability of AdGuard/uBlock Origin.",
     "homeUrl": "https://github.com/Yuki2718/adblock",
     "id": 2095,
     "issuesUrl": "https://github.com/Yuki2718/adblock/issues",
     "licenseId": 28,
-    "name": "Anti-scam enhancer for 280blocker ABP format list"
+    "name": "Anti-scam enhancer for 280blocker adblock filter"
+  },
+  {
+    "description": "Removes blogroll (feed-style mutual links) on Japanese sites. Included in Yuki's uBlock Japanese filters - Annoyances",
+    "homeUrl": "https://github.com/Yuki2718/adblock",
+    "id": 2096,
+    "issuesUrl": "https://github.com/Yuki2718/adblock/issues",
+    "licenseId": 28,
```

Comment:
licenseId 28 is CC0 1.0.  Your lists, except of Anti-scam enhancer, should be provided under CC BY-SA 4.0.

Reply:
Thx for pointing out my mistake!
@collinbarrett Could you change the license?

Reply:
captured in #2345 for me or somebody else to address as we have time.